pkg/doc/did/doc.go

```diff
@@ -201,17 +210,17 @@ type rawDoc struct {
 	Authentication []interface{}            `json:"authentication,omitempty"`
 	Created        *time.Time               `json:"created,omitempty"`
 	Updated        *time.Time               `json:"updated,omitempty"`
-	Proof          *Proof                   `json:"proof,omitempty"`
+	Proof          []interface{}            `json:"proof,omitempty"`
 }
 
 // Proof is cryptographic proof of the integrity of the DID Document
 type Proof struct {
-	Type           string     `json:"type,omitempty"`
-	Created        *time.Time `json:"created,omitempty"`
-	Creator        string     `json:"creator,omitempty"`
-	SignatureValue string     `json:"signatureValue,omitempty"`
-	Domain         string     `json:"domain,omitempty"`
-	Nonce          string     `json:"nonce,omitempty"`
+	Type       string     `json:"type,omitempty"`
+	Created    *time.Time `json:"created,omitempty"`
+	Creator    string     `json:"creator,omitempty"`
+	ProofValue string     `json:"proofValue,omitempty"`
```

Comment:
Should ProofValue and Nonce be []byte?
(Seems odd to see a string.)

Reply:
This was existing struct. I changed ProofValue and Nonce to []byte. Done.